core-features/blog-env-postgresql/index.spec.js

```diff
@@ -0,0 +1,160 @@
+const assert = require('assert')
+const getPort = require('get-port')
+const tcpProxy = require('node-tcp-proxy')
+const { Client } = require('pg')
+const { PrismaClient, PrismaClientValidationError } = require('@prisma/client')
+
+describe('Prisma client and postgres works with a proxy and flakyness', () => {
+  let requests = []
```

Comment:
`let` could be replaced with `const` I think?

Reply:
If it doesn't change then yes